pkg/gatewayserver/config.go

```diff
@@ -29,9 +29,9 @@ type UDPConfig struct {
 	Listeners  map[string]string `name:"listeners" description:"Listen addresses with (optional) fallback frequency plan ID for non-registered gateways"`
 }
 
-// BasicStationConfig defines the Basic Station configuration of the Gateway Server.
+// BasicStationConfig defines the LoRa Basics Station configuration of the Gateway Server.
```

Comment:
I can also change this to LBS but we can't change the json tag for obvious reasons.